src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -484,27 +486,46 @@ export default class DefaultAudioVideoController
     this.meetingSessionContext.enableSimulcast = this.enableSimulcast;
 
     if (this.enableSimulcast) {
```

Comment:
Hmm, it'd be nice if builders didn't have to set `enableSimulcastForChromiumBasedBrowsers` if they were passing in a uplink policy that implemented `SimulcastUplinkPolicy`.

Reply:
They shouldn’t need to; enableSimulcast should only be relevant if they don’t explicitly provide a policy.

Reply:
Add enabling simulcast if passing in simulcast policy.